### ISSUE TITLE
Tag lookup fails when git refs contain commas

### ISSUE THREAD

**Issue by pbjc** [mtkennerly/dunamai#13]
Since dunamai parses git tags by running `git log --simplify-by-decoration --topo-order --decorate=full HEAD "--format=%H%d"` and splitting on commas, it will fail with an unhelpful error if any tag contains a comma. For example, with the following tag list:

```$ git log --simplify-by-decoration --topo-order --decorate=full HEAD "--format=%H%d"
b8847fb838aa45b474537e0c3f8aa5f385cd3ed0 (HEAD -> refs/heads/master, tag: refs/tags/0.0.1,, tag: refs/tags/0.0.1)
```

Running `poetry version` with any poetry-dynamic-versioning config results in this error:
```$ poetry version

  RuntimeError

  Unable to determine commit offset for ref refs/tags/0.0.1, in data: {'refs/tags/0.0.1': 0}

  at ~/.local/pipx/venvs/poetry/lib/python3.6/site-packages/dunamai/__init__.py:233 in commit_offset
       229│             return self.tag_topo_lookup[self.fullref]
       230│         except KeyError:
       231│             raise RuntimeError(
       232│                 "Unable to determine commit offset for ref {} in data: {}".format(
    →  233│                     self.fullref, self.tag_topo_lookup
       234│                 )
       235│             )
       236│
       237│     @property
```

It's unclear what the issue is from a user perspective, and it's not obvious that the comma is a part of the tag here. It might be tricky to parse tags containing commas properly, but it would be nice if the error message at least was more helpful since we know that commas can trip things up  with the way parsing is currently done.

(it seems that this behavior started happening in 1.5.1)

**Comment by mtkennerly** [comment]
Thanks for reporting this! Somehow, I didn't realize that commas were valid in a tag. I think it should be easy enough to fix by splitting on `", "` instead of `","` since spaces are not valid in tags, but I'll add some tests to make sure.

**Comment by mtkennerly** [comment]
This is now fixed in v1.5.4.

**Comment by pbjc** [comment]
Thanks for the quick turnaround!
